lib/tasks/era_committees.rake

```diff
@@ -0,0 +1,35 @@
+namespace :era_committee do
+  desc 'Populate the era committees for players'
+  task populate: :environment do
+    Player.connection.execute %{
+      WITH era_stats AS (
+        SELECT SUM(season_stats.hall_rating) AS rating,
+        player_id,
+        CASE WHEN year <= 1949 THEN 'early_baseball'
+          WHEN year > 1949 AND year <= 1969 THEN 'golden_days'
+          WHEN year > 1969 AND year <= 1987 THEN 'modern_baseball'
+          ELSE 'todays_game'
+          END AS era,
+
+        CASE WHEN year <= 1949 THEN 1
+          WHEN year > 1949 AND year <= 1969 THEN 2
+          WHEN year > 1969 AND year <= 1987 THEN 3
+          ELSE 4
+          END AS era_tiebreaker
+
+        FROM season_stats
+        JOIN players ON players.id = season_stats.player_id
+        WHERE players.eligibility = 'eligible'
```

Comment:
This looks really great, guys. Didn't run it yet, just read through the code. I think I missed one requirement, but I think it's as easy as a single line. Hank Aaron shouldn't be eligible for a committee since he's in the HOF. So I'm guessing I want to add:

`WHERE players.hof = false`

after this line. Sound reasonable?

Now, to add this, do I just run

`rake era_committee` ?

Thanks!